FusionIIIT/applications/otheracademic/views.py

```diff
@@ -1,8 +1,30 @@
+# from django.shortcuts import render
+# from django.contrib import messages
+# from django.shortcuts import render, get_object_or_404, redirect
+# # from .models import File, Tracking
+# from applications.globals.models import ExtraInfo, HoldsDesignation, Designation
+# from django.template.defaulttags import csrf_token
+# from django.http import HttpResponse, HttpResponseRedirect, JsonResponse
+# from django.contrib.auth.decorators import login_required
+# from django.db import IntegrityError
+# from django.core import serializers
+# from django.contrib.auth.models import User
+# from timeit import default_timer as time
+# from notification.views import office_module_notif,file_tracking_notif
+# from django.http import JsonResponse
+# from django.views.decorators.csrf import csrf_exempt
+# from .models import LeaveFormTable,BonafideFormTableUpdated,GraduateSeminarFormTable,AssistantshipClaimFormStatusUpd,NoDues
+# from django.shortcuts import render, get_object_or_404
+# from datetime import date 
+# from applications.filetracking.models import *
+# from applications.filetracking.sdk.methods import *
```

Comment:
why these comments? Please remove the code that is not longer in use.

Reply:
Ok i wil remove all the comments